Sources/protoc-gen-swift/main.swift

```diff
@@ -27,8 +27,10 @@ import PluginLibrary
 enum GenerationError: Error {
   /// Raised for any errors reading the input
   case readFailure
-  /// Raise when parsing the parameter string and found an unknown key
+  /// Raised when parsing the parameter string and found an unknown key
   case unknownParameter(name: String)
+  /// Raised when a parameter was giving an invalid value
+  case invalidParameterValue(name: String, value: String)
```

Comment:
I debated that one, but I wasn't sure if there was an easy way to collect that from the `enum` to avoid having a second list to update.  I'm sorta hoping the docs alone will cover it.